HomeworkTwelve/src/main/java/Car/CarArrayExample.java

```diff
@@ -0,0 +1,76 @@
+package Car;
+
+import Car.Engines.ElectricEngine;
+import Car.Engines.PetrolEngine;
+import Car.Enums.Fuel;
+import Car.Interfaces.IEngine;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class CarArrayExample {
+
+    public static void main(String[] args) {
```

Comment:
you don't need this class in current homework, because you need to test CarArray methods